liquibase-maven-plugin/src/main/java/org/liquibase/maven/plugins/LiquibaseRollbackOneChangeSetMojo.java

```diff
@@ -0,0 +1,134 @@
+import liquibase.command.CommandFactory;
+import liquibase.command.LiquibaseCommand;
+import liquibase.database.Database;
+import liquibase.exception.LiquibaseException;
+import org.apache.maven.plugin.MojoFailureException;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ *
+ * Invokes Liquibase targeted rollback
+ *
+ * @goal rollbackOneChangeSet
+ *
+ */
+public class LiquibaseRollbackOneChangeSetMojo extends AbstractLiquibaseChangeLogMojo {
+    /**
+     *
+     * The change set ID to rollback
+     *
+     * @parameter property="liquibase.changeSetId"
+     *
+     */
+    protected String changeSetId;
+
+    /**
+     *
+     * The change set author to rollback
+     *
+     * @parameter property="liquibase.changeSetAuthor"
+     *
+     */
+    protected String changeSetAuthor;
+
+    /**
+     *
+     * The path to the changelog where this
+     * change set to rollback lives
+     *
+     * @parameter property="liquibase.changeSetPath"
+     *
+     */
+    protected String changeSetPath;
+
+    /**
+     *
+     * Required flag for RollbackOneChangeSet
+     *
+     * @parameter property="liquibase.force"
+     *
+     */
+    protected String force;
+
+    /**
+     *
+     * The path to a rollback script
+     *
+     * @parameter property="liquibase.rollbackScript"
+     *
+     */
+    protected String rollbackScript;
+
+    @Override
+    protected void checkRequiredParametersAreSpecified() throws MojoFailureException {
+      super.checkRequiredParametersAreSpecified();
+      checkRequiredRollbackParameters();
+    }
+
+    protected void checkRequiredRollbackParameters() throws MojoFailureException {
+
+      String message = "Cannot specify multiple rollbackXXX options, please select only"
+                       + " one of rollbackTag, rollbackCount, rollbackDate.";
+
+    }
+
+    @Override
+    protected void printSettings(String indent) {
+      super.printSettings(indent);
+        getLog().info(indent + "Change Set ID:     " + changeSetId);
+        getLog().info(indent + "Change Set Author: " + changeSetAuthor);
+        getLog().info(indent + "Change Set Path:   " + changeSetPath);
+        getLog().info(indent + "Rollback script:   " + rollbackScript);
+    }
+
+    @Override
+    protected void performLiquibaseTask(Liquibase liquibase) throws LiquibaseException {
+        //
+        // Check the Pro license
+        //
+        if (! hasProLicense()) {
+            return;
+        }
+        Database database = liquibase.getDatabase();
+        LiquibaseCommand liquibaseCommand = (CommandFactory.getInstance().getCommand("rollbackOneChangeSet"));
+        AbstractSelfConfiguratingCommand configuratingCommand = (AbstractSelfConfiguratingCommand)liquibaseCommand;
+        Map<String, Object> argsMap = getCommandArgsObjectMap(liquibase);
+        ChangeLogParameters clp = new ChangeLogParameters(database);
+        argsMap.put("changeLogParameters", clp);
+        if (force != null) {
```

Comment:
Since they have to pass force=true as an argument, we should make sure they specified "true" not just anything